# a list of joint goals
control_msgs/FollowJointTrajectoryGoal[] waypoints

# the number of waypoints to visualize in RViz (0 will show all, 1 will show just the last/goal point, any other number will evenly distribute between start and goal)
int16 num_visualizaton_points
